src/transport/http2.rs

```diff
@@ -142,22 +135,22 @@ impl<In: RpcMessage, Out: RpcMessage> ServerChannel<In, Out> {
                         match chunk {
                             Ok(chunk) => match bincode::deserialize::<In>(chunk.as_ref()) {
                                 Ok(msg) => {
-                                    log!("server got msg {:?}", msg);
+                                    event!(Level::TRACE, "server got msg {:?}", msg);
```

Comment:
I think we might not want to put the msg in the event even in trace mode. Maybe just the length of the deserialized blob?

Having Debug on messages is a double edged sword since formatting giant messages can really mess with your performance without you noticing.